.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       - name: Run E2E Test ${{ matrix.suite }}
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite=${{ matrix.suite }} --prod_env
         env:
-          VIDEO_RECORDING_IS_ENABLED: 0
+          VIDEO_RECORDING_IS_ENABLED: 1
```

Comment:
ARe we oin to keep this as is?

Reply:
No, I will change this before merging the PR. I did this to debug the test in the CI.

Reply:
Generally, we assign reviewers when the PR changes are ready to be merged. Can you please fix it?